src/prime_rl/trainer/rl/train.py

```diff
@@ -116,10 +121,23 @@ def train(config: RLTrainerConfig):
     logger.info(f"Initializing optimizer ({config.optim})")
     logger.info(f"Using `{config.loss.ratio_type}` importance ratio ({config.loss})")
 
-    optimizer = setup_optimizer(config.optim, model, parallel_dims.world_mesh["dp_shard_cp"])
+    optimizer = setup_multi_optimizer(
```

Comment:
if we don't have mutli run we should not be instantiating multi optimizer imo. We should probably have a config to enable multi run in which case we trigger the multi optimizer, same for multi scheduler. 

And I would rather call them multi_lora_optimizer to make it clear